client/scss/components/_a11y-result.scss

```diff
@@ -43,6 +48,15 @@
   height: theme('spacing.[7.5]');
   width: theme('spacing.[7.5]');
 
+  .form-side--checks & {
+    background-color: theme('colors.surface-button-outline-hover');
+    margin-top: theme('spacing.[2.5]');
+    margin-inline-end: 0;
+    padding: theme('spacing.1');
+    gap: theme('spacing.[1.5]');
+    width: auto;
```

Comment:
If the selector is very long, it looks a bit ugly. This is an issue even in older versions, but do you think it's worth fixing?

<img src="https://github.com/user-attachments/assets/bb10a128-a753-497d-87aa-3779326b4cad" alt="the selector text overflowing the button container" width="400"/>

I was able to fix it by adding the following:

```suggestion
    width: auto;
    max-width: auto;
```

and then adding

```scss
.w-a11y-result__selector-text {
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
}
```

(and the class to the corresponding `<span>`)

Result:

<img alt="the selector text clipped with an ellipsis at the end" src="https://github.com/user-attachments/assets/b10dd47c-d33f-4b3e-8b7b-ddd04c594aa2" width="400" />